### ISSUE TITLE
Table: Accessibility issues with reading Table row/header using JAWS screen reader software

### ISSUE THREAD

**Issue by manishcharsap** [SAP/ui5-webcomponents-react#5751]
### Describe the bug

When accessing the row in the table, the contents/ information’s present within the rows and columns in the “Keys” table is not announced

### Reproduction steps

1. Open JAWS screen reader software.
2. Go to the table and access table row/header using tab. 
3. 
...

### Expected Behaviour

All the contents / information’s should be conveyed to the user. Here in the table the information/ contents present in the rows and column should be announced to the user when the rows and columns are accessed, and the correct row and the column and row count information of the table should be read out correctly to the screen reader user when accessed.

### Screenshots or Videos

<img width="924" alt="PTBS-38912-tabel-issue" src="https://github.com/SAP/ui5-webcomponents-react/assets/103047215/4678ce9c-80c4-4b10-90c7-0ec9d3c1d61b">

### UI5 Web Components for React Version

1.19.0

### UI5 Web Components Version

1.17.0

### Browser

Chrome

### Operating System

windows

### Organization

SAP Labs India

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.
